test/spec/unit/pbjs_api_spec.js

```diff
@@ -779,6 +779,24 @@ describe('Unit: Prebid Module', function () {
       adaptermanager.callBids.restore();
     });
 
+    it('should log an error if a bidder is used that does not exist', () => {
```

Comment:
This test should go in adapterManager_spec. You can use adaptermanager.callBids function to test. 
#1508 has some good arguments to avoid $$PREBID_GLOBAL$$ in unit tests
 
Also stubs should be in beforeEach loop because if this test fails to avoid `logError` wont be restored.